pinot-common/src/main/java/com/linkedin/pinot/common/metrics/ServerGauge.java

```diff
@@ -30,7 +30,8 @@
   LAST_REALTIME_SEGMENT_INITIAL_CONSUMPTION_DURATION_SECONDS("seconds", false),
   LAST_REALTIME_SEGMENT_CATCHUP_DURATION_SECONDS("seconds", false),
   LAST_REALTIME_SEGMENT_COMPLETION_DURATION_SECONDS("seconds", false),
-  KAFKA_PARTITION_OFFSET_LAG("messages", false);
+  KAFKA_PARTITION_OFFSET_LAG("messages", false),
+  RUNNING_QUERIES("runningQueries", false);
```

Comment:
Discussed. I incorrectly reported that this is server level. We want to it per table table for multitenant clusters